validation/longhorn/schemas/pit_schemas.md

```diff
@@ -0,0 +1,399 @@
+| 2 | Navigate to target cluster in Rancher dashboard   |      | Cluster view displays                                       |
+| 3 | Click "Apps" in the left navigation menu          |      | Apps section opens                                          |
+| 4 | Click "Charts" to access the application catalog  |      | Charts catalog displays                                     |
+| 5 | Search for "Longhorn" in the search bar           |      | Longhorn chart appears in results                           |
+| 6 | Select the Longhorn chart from the results        |      | Longhorn installation page opens                            |
+| 7 | Review default settings and click "Install"       |      | Installation begins                                         |
+| 8 | Monitor installation progress in the Apps section |      | Installation completes within 10 minutes                    |
+| 9 | Verify all Longhorn pods are in Running state     |      | All pods show "Running" status in longhorn-system namespace |
+| 10 | Check Longhorn UI accessibility through Rancher |      | Longhorn UI opens when clicking Longhorn button  |
+| 11 | Verify default storage class creation     |      | `longhorn` storage class exists and is available |
+
+### Longhorn Installation with Custom Configuration
+
+Test Longhorn installation with custom settings including replica count, data path, and node selection through Rancher interface.
+
+| Step Number | Action | Data | Expected Result                             |
+|-------------|--------|--------|----------------------------------------------|
+| 1           | Access Longhorn installation chart in Rancher Apps | | Longhorn installation page displays         |
+| 2           | Click "Longhorn Default Settings" to expand configuration | | Configuration options become visible         |
+| 3           | Set Default replica count to `2` | | Setting accepts value of 2                   |
+| 4           | Set Default data path to `/var/lib/longhorn-custom` |  | Custom path is configured                    |
+| 5           | Set Storage over-provisioning percentage to `150` |  | Over-provisioning set to 150%                |
+| 6           | Enable "Create default disk only on labeled node" | | Option is enabled                            |
+| 7           | Label worker nodes with `node.longhorn.io/create-default-disk=true` | | Nodes are properly labeled                  |
+| 8           | Complete installation with custom configuration | | Installation succeeds with custom settings   |
+| 9           | Verify custom settings in Longhorn UI | | Settings reflect custom configuration        |
+| 10          | Create test volume and verify replica count | | Volume created with 2 replicas               |
+| 11          | Check custom data path utilization on nodes | | `/var/lib/longhorn-custom` directory is used |
+
+### Longhorn UI Access Through Rancher
```

Comment:
isn't this verifying longhorn is installed? Or in other words, what's the prerequisite to this test?